README.md

```diff
@@ -97,12 +97,3 @@ It's a miracle. Get it at http://www.graylog2.org/
 ## Installation
 
     npm install graylog2
-
-## License
```

Comment:
The MIT license isn't a "do-whatever-you-want-for-free" license... it requires attribution, and maintaining the license for any significant portion of the software that's copied. "do-whatever-you-want-for-free" is public domain dedication, or WTFPL.

Plus it's covered in package.json & LICENSE file now... so it's pretty redundant.